website/cue/reference/releases/0.42.0.cue

```diff
@@ -0,0 +1,379 @@
+package metadata
+
+releases: "0.42.0": {
+	date:     "2024-10-16"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		The Vector team is pleased to announce version 0.42.0!
+
+		This release contains a number of enhancements and fixes as listed below.
+
+		There are no breaking changes or deprecations with this release and so no upgrade guide.
+		"""
+
+	changelog: [
+		{
+			type: "feat"
+			description: """
+				VRL was updated to v0.19.0. This includes the following changes:
+
+				### Breaking Changes & Upgrade Guide
+
+				- The multi-line mode of the `parse_groks` VRL function is now enabled by default.
+				Use the `(?-m)` modifier to disable this behaviour. (https://github.com/vectordotdev/vrl/pull/1022)
+
+				### Enhancements
+
+				- The `keyvalue` grok filter is extended to match Datadog implementation. (https://github.com/vectordotdev/vrl/pull/1015)
+
+				### Fixes
+
+				- The `parse_xml` function now doesn't add an unnecessary `text` key when processing single nodes. (https://github.com/vectordotdev/vrl/pull/849)
+				- `parse_grok` and `parse_groks` no longer require field names containing a hyphen (e.g. `@a-b`) to be quoted.
+				- The function `match_datadog_query` doesn't panic if an invalid path is passed, instead it returns an error. (https://github.com/vectordotdev/vrl/pull/1031)
+				- The `parse_ruby_hash` parser is extended to match Datadog implementation. Previously it would parse the key in `{:key => "value"}` as `:key`, now it will parse it as `key`. (https://github.com/vectordotdev/vrl/pull/1050)
```

Comment:
```suggestion
				- The `parse_ruby_hash` parser is extended to match the Datadog implementation. Previously it would parse the key in `{:key => "value"}` as `:key`, now it parses it as `key`. (https://github.com/vectordotdev/vrl/pull/1050)
```

Reply:
An additional suggestion per the [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense)